index.html

```diff
@@ -158,156 +161,70 @@ <h1 class="work-sample-2"> Multy-Post Stories </h1>
+			<button class="see" type="button" onclick="myFunction(this)" id="4">See Project</button>
 		</div>
-		<div class="cards">
-			<h1 class="work-sample"> Profesional Art Printing Data </h1>
-			<p class="work-sample-text">
-				A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
-			</p>
-			<ul class="tags">
-				<li class="tag">html</li>
-				<li class="tag">bootstrap</li>
-				<li class="tag">ruby</li>
-			</ul>
-			<a class="see" href="#"> See Project</a>
+		<div class="cards" id="cards-sample-7">
+			<button class="see" type="button" onclick="myFunction(this)" id="5">See Project</button>
 		</div>
-		<div class="cards">
-			<h1 class="work-sample"> Profesional Art Printing Data </h1>
-			<p class="work-sample-text">
-				A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
-			</p>
-			<ul class="tags">
-				<li class="tag">html</li>
-				<li class="tag">bootstrap</li>
-				<li class="tag">ruby</li>
-			</ul>
-			<a class="see" href="#"> See Project</a>
+		<div class="cards" id="cards-sample-8">
+			<button class="see" type="button" onclick="myFunction(this)" id="6">See Project</button>
 		</div>
-		<div class="cards">
-			<h1 class="work-sample"> Profesional Art Printing Data </h1>
-			<p class="work-sample-text">
-				A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
-			</p>
-			<ul class="tags">
-				<li class="tag">html</li>
-				<li class="tag">bootstrap</li>
-				<li class="tag">ruby</li>
-			</ul>
-			<a class="see" href="#"> See Project</a>
+		<div class="cards" id="cards-sample-9">
+			<button class="see" type="button" onclick="myFunction(this)" id="7">See Project</button>
 		</div>
-		<div class="cards">
-			<h1 class="work-sample"> Profesional Art Printing Data </h1>
-			<p class="work-sample-text">
-				A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
-			</p>
-			<ul class="tags">
-				<li class="tag">html</li>
-				<li class="tag">bootstrap</li>
-				<li class="tag">ruby</li>
-			</ul>
-			<a class="see" href="#"> See Project</a>
+		<div class="cards" id="cards-sample-10">
+			<button class="see" type="button" onclick="myFunction(this)" id="8">See Project</button>
 		</div>
-		<div class="cards">
-			<h1 class="work-sample"> Profesional Art Printing Data </h1>
-			<p class="work-sample-text">
-				A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
-			</p>
-			<ul class="tags">
-				<li class="tag">html</li>
-				<li class="tag">bootstrap</li>
-				<li class="tag">ruby</li>
-			</ul>
-			<a class="see" href="#"> See Project</a>
+		<div class="cards" id="cards-sample-11">
+			<button class="see" type="button" onclick="myFunction(this)" id="9">See Project</button>
 		</div>
 	</div>
```

Comment:
- [ ] Kindly be noticed that the `See Project` button for some projects is not visible please fix that.

![Screenshot from 2022-06-24 16-27-47](https://user-images.githubusercontent.com/33752735/175556956-199cdbc7-a1ab-45a3-97bd-3f14b8f47262.png)